includes.sh

```diff
@@ -112,11 +113,29 @@ cona() {
     : 'CodeNAme'
     if [[ $GITHUB_REPOSITORY ]]; then
         echo "${GITHUB_REPOSITORY##*/}"
+    elif [[ $VIRTUAL_ENV ]]; then
+        basename "${VIRTUAL_ENV%/.venv}"
     else
-        echo "${VIRTUAL_ENV:-$PWD}" | sed -E 's,.*/([^/]+)(/\.?venv)?,\1,'
+        basename "$PWD"
     fi
 }
 
+check_mailmap() {
```

Comment:
Why not use thsi? https://github.com/jumanjihouse/pre-commit-hooks#check-mailmap

Reply:
I'm trying to cut back on the number of git repositories that pre-commit fetches and I think this implementation is simpler.